server/reload.go

```diff
@@ -1269,6 +1269,7 @@ func (s *Server) reloadAuthorization() {
 
 	// We will double check all JetStream configs on a reload.
 	if checkJetStream {
+		s.getJetStream().clearResources()
```

Comment:
I could not figure out what the issue was looking at this change. I now understand that the reserved memory/store was failing the sufficientResources() check. I would strongly recommend that we log ERRor that we may get in the `s.GlobalAccount().EnableJetStream(nil)` call.. that would have made the diagnostic easier.

Reply:
ok will add.